import sqlite3
from sqlite3 import Error
 
 
conn = sqlite3.connect('first_database.db')